Ugee_EX07/index.md

```diff
@@ -0,0 +1,18 @@
+---
+VID: "28bd"
+PID: "0071"
+vendor: Ugee
+product: EX07
+image: Ugee_EX07
+report_rate: 220
+working_area:
+    width: 8
+    height: 5
+    resolution: 5080
+pen:
+    pressure_levels: 2048
+frame_controls: 6 buttons
+sold_as:
+supported: false
+supported_in:
```

Comment:
Same here about support, only it's been supported since [v7](https://github.com/DIGImend/digimend-kernel-drivers/releases/tag/v7), so `supported_in` should say `digimend: ">= 7"`. See https://raw.githubusercontent.com/DIGImend/tablets/master/UC-Logic_WP5540U_v2/index.md for an example.